packages/manager/modules/enterprise-cloud-database/src/service/details/backups/recovery/recovery.controller.js

```diff
@@ -43,4 +46,14 @@ export default class EnterpriseCloudDatabaseServiceDetailsBackupsRecoveryCtrl {
       ))
       .finally(() => { this.isLoading = false; });
   }
+
+  validDate() {
+    const selectedDateTime = moment(`${this.selectedDate} ${this.selectedTime}`);
+    if (selectedDateTime.isAfter(this.minDate) && selectedDateTime.isBefore(this.maxDate)) {
+      this.isValidDate = true;
+      return true;
+    }
+    this.isValidDate = false;
+    return false;
```

Comment:
you can `return this.isValidDate`, and remove the return line inside the if block